src/jats/jats-exporter.ts

```diff
@@ -424,10 +433,14 @@ export class JATSExporter {
     }
   }
 
-  protected buildFront = (doi?: string, id?: string, links?: Links) => {
-    const manuscript = findManuscript(this.modelMap)
-    const titles = findTitles(this.modelMap)
-
+  protected buildFront = (
+    manuscript: Manuscript,
```

Comment:
Do we actually need this? We should be able to derive any information directly from the manuscript node.

Reply:
There are attributes that are available in the model but missing in the manuscript node, e.g. `figureCount`, `acceptanceDate`, and many more. We can update the manuscript node to have these attributes.

Reply:
We should be able to derive this info from the document, right? No need to store it explicitly.